i18n/en.json

```diff
@@ -284,7 +269,6 @@
 	"num-edits-size-interval": "Number of edits in that size change interval, in bytes",
 	"num-hours": "{{PLURAL:$1|hour|hours}}",
 	"num-minutes": "{{PLURAL:$1|minute|minutes}}",
-	"num-months": "{{PLURAL:$1|month|months}}",
```

Comment:
Let's keep this along with `num-weeks` and `num-years` for https://phabricator.wikimedia.org/T406182